package.json

```diff
@@ -1169,19 +1163,19 @@
         "when": "confluent.ccloudConnectionAvailable && confluent.flinkStatementsPoolSelected && confluent.flinkStatementsSearchApplied"
       },
       {
-        "view": "confluent-flink-artifacts",
-        "contents": "No Flink compute pools available.\n[Connect to Confluent Cloud](command:confluent.connections.ccloud.signIn)",
+        "view": "confluent-flink-database",
+        "contents": "No Flink databases (CCloud Kafka clusters) available.\n[Connect to Confluent Cloud](command:confluent.connections.ccloud.signIn)",
```

Comment:
Some people might think this copy is heavy-handed but I'm a fan of the clarity it offers. If you don't hammer me over the head with the duality I won't remember what Kafka entity corresponds to what Flink entity.